README.md

```diff
@@ -2,96 +2,92 @@
+[downloads-image]:http://img.shields.io/npm/dm/redux-mock-store-await-actions.svg
+[travis-url]:https://travis-ci.org/moxystudio/redux-mock-store-await-actions
+[travis-image]:http://img.shields.io/travis/moxystudio/redux-mock-store-await-actions/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/redux-mock-store-await-actions
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/redux-mock-store-await-actions/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/redux-mock-store-await-actions
+[david-dm-image]:https://img.shields.io/david/moxystudio/redux-mock-store-await-actions.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/redux-mock-store-await-actions?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/redux-mock-store-await-actions.svg
+[greenkeeper-image]:https://badges.greenkeeper.io/moxystudio/redux-mock-store-await-actions.svg
 [greenkeeper-url]:https://greenkeeper.io
 [conventional-commits-url]:https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg
 
-> Waits for actions to be dispatched until a timeout expires.
+> Waits for actions dispatched asynchronously until a timeout expires.
 
 
 ## Installation
 
-`$ npm install redux-wait-actions --save-dev`
+`$ npm install redux-mock-store-await-actions --save-dev`
 
 
 ## Motivation
 
-This module shall be used only for testing purposes to wait for the dispatch of actions that are dispatched in an asynchronous fashion by thunks.
-
-When `thunk` dispatches various actions asynchronously, the following test will fail:
+When `asyncActionCreator` dispatches various actions asynchronously, the following test will fail:
 
 ```js
-store.dispatch(thunk(...));
-
+store.dispatch(asyncActionCreator());
 expect(store.getActions()).toEqual(expect.arrayContaining(expectedActions));
 ```
 
 To solve this, one can use `setTimeout` explicitly in each test:
 
 ```js
-store.dispatch(thunk(...));
-
+store.dispatch(asyncActionCreator());
 setTimeout(() => {
 	expect(store.getActions()).toEqual(expect.arrayContaining(expectedActions));
 }, 50);
 ```
 
-To get around this ugliness, Redux Wait Actions sets the timeout for you and returns a `Promise` to wait for actions dispatch and handle timeouts.
-
+To get around this ugliness, `redux-mock-store-await-actions` sets the timeout for you and returns a `Promise` to await for dispatch of specified actions and handle timeout.
 
 ## Usage
 
-### Example #1: single action
-
-```js
-const waitForActions = require('redux-wait-actions');
+**NOTE:** This module only works with [redux-mock-store](https://github.com/arnaudbenard/redux-mock-store) and shall only be used for testing purposes. Support for real redux store is not provided. 
 
-const action = { type: 'action' };
-
-const promise = waitForActions(store, action);
-
-store.dispatch(action);
+### Example #1: action types
 
+```js
+const promise = awaitForActions(store, ['action-1', action-2', ...]);
```

Comment:
We can't have invalid code on the examples otherwise the formatting will be messed up.

Reply:
Also I would make some real-world scenarios, simulating that we are testing a login.

I would create mock store + dispatch a `login('andre-lima', '1234')` and then test if `LOGIN_START` & `LOGIN_SUCCESS` occur.